analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.CompoundAssignment.cs

```diff
@@ -63,13 +83,13 @@ public void CompoundAssignment_SelfUpdate()
             Tag("Value", value);
             """;
         var validator = SETestContext.CreateCS(code).Validator;
-        validator.TagValue("Result").Should().HaveOnlyConstraints(ObjectConstraint.NotNull);
-        validator.TagValue("Value").Should().HaveOnlyConstraints(ObjectConstraint.NotNull);
+        validator.TagValue("Result").Should().HaveOnlyConstraints(ObjectConstraint.NotNull, NumberConstraint.From(42));
+        validator.TagValue("Value").Should().HaveOnlyConstraints(ObjectConstraint.NotNull, NumberConstraint.From(42));
     }
 
     [DataTestMethod]
-    [DataRow("+=")]
-    [DataRow("-=")]
+  // [DataRow("+=")]
```

Comment:
The plan is to merge on this https://github.com/SonarSource/sonar-dotnet/pull/7923
and then fix this test.